<launch>

  
  <node pkg="imu_driver" type="mick_imu" name="wit_imu" output="screen">
 	  <param name="dev" value="/dev/ttyUSB0" type="str" />
	  <param name="buad" value="115200" type="int" />
	  <param name="time_out" value="1000" type="int" />
	  <param name="hz" value="100" type="int" />
	  <param name="show_message" value="1" type="int" />
  </node>

 

</launch>
